tests/python/PyFlakesBearTest.py

```diff
@@ -0,0 +1,15 @@
+from bears.python.PyFlakesBear import PyFlakesBear
+from tests.LocalBearTestHelper import verify_local_bear
+
+good_file = """
+print("Hi")
+"""
+
+bad_file = """
+import os
+print("Hi")
+"""
+
+PyFlakesBear = verify_local_bear(PyFlakesBear,
+                                 valid_files=(good_file,),
+                                 invalid_files=(bad_file,))
```

Comment:
`bad_file` should only be level info messages; does `verify_local_bear` consider level info messages to be an invalid file?